templates/contacts/index.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'contacts/base.html' %}
+{% block title %}
+    Index
+{% endblock %}
+{% block content %}
+
+    <div>
+        <h2>Your Contacts</h2>
+    </div>
+{#    <button type="button" class="btn btn-link btn-lg" data-toggle="modal" data-target="#add_contact_modal">Add + </button>#}
+    <a href="/insert" class="btn btn-link btn-lg" >Add + </a>
+    <table class="table" id="table_contact">
+        <thead>
+            <tr>
+                <th>ID</th>
+                <th>Name</th>
+                <th>Email</th>
+                <th>Number</th>
+                <th>Birthday</th>
+                <th>Note</th>
+                <th>Actions</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>1</td>
```

Comment:
Why is this entry hard coded?

Reply:
i use loop counter to show a sequence 

forloop.counter